locale/pt-br/texts.po

```diff
@@ -1586,16 +1586,99 @@ msgstr "Ok, fechar"
 msgid "Error while sending create token transaction."
 msgstr "Erro ao enviar transação de criação de token."
 
-#: src/components/Reown/SignMessageModal.js:63
+#: src/components/Reown/InsufficientFundsModal.js:37
+msgid "Insufficient funds to complete the transaction."
+msgstr ""
```

Comment:
Missing translation (there are some missing)

Reply:
Added the missing translations